webauthn-authenticator-rs/src/cable/discovery.rs

```diff
@@ -2,7 +2,6 @@
 #[cfg(doc)]
 use crate::stubs::*;
 
-use num_traits::ToPrimitive;
```

Comment:
If you update `num-derive` from `0.3` to `0.4`, all of the non-local `impl` warnings go away. :)

Reply:
... ugh, seriously? I tried updating things and all the conditional compilation shit must have hidden this. 😢